framework/integration/client/nodeconfig/crud.go

```diff
@@ -0,0 +1,58 @@
+// +build k8srequired
+
+package nodeconfig
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/microerror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (c Client) CreateObject(namespace string, obj interface{}) (interface{}, error) {
+	nodeConfig, err := toCustomObject(obj)
+	if err != nil {
+		return nil, err
+	}
+	createNodeConfig, err := c.g8sClient.CoreV1alpha1().NodeConfigs(namespace).Create(&nodeConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	return createNodeConfig, nil
+}
+
+func (c Client) DeleteObject(name, namespace string) error {
+	err := c.g8sClient.CoreV1alpha1().NodeConfigs(namespace).Delete(name, nil)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (c Client) GetObject(name, namespace string) (interface{}, error) {
+	nodeConfig, err := c.g8sClient.CoreV1alpha1().NodeConfigs(namespace).Get(name, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	return nodeConfig, nil
+}
+
+func (c Client) UpdateObject(namespace string, obj interface{}) (interface{}, error) {
+	return nil, nil
```

Comment:
This is noop because no test uses it yet, I will implement these once I need them.